test/integration/auth/node_test.go

```diff
@@ -267,6 +279,36 @@ func TestNodeAuthorizer(t *testing.T) {
 			return err
 		}
 	}
+	setNode2ConfigSource := func(client clientset.Interface) func() error {
+		return func() error {
+			node2, err := client.Core().Nodes().Get("node2", metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			node2.Spec.ConfigSource = &api.NodeConfigSource{
+				ConfigMapRef: &api.ObjectReference{
+					Namespace: "ns",
+					Name:      "myconfigmapconfigsource",
+					// validation just requires UID to be non-empty and it isn't necessary for GET,
+					// so we just use a bogus one for the test
+					UID: "uid",
```

Comment:
if you have valid data from creating the configmap above, go ahead and set it here